includes/embeds/class-amp-twitter-embed.php

```diff
@@ -46,7 +53,9 @@ class AMP_Twitter_Embed_Handler extends AMP_Base_Embed_Handler {
 	 * Registers embed.
 	 */
 	public function register_embed() {
-		add_shortcode( 'tweet', [ $this, 'shortcode' ] ); // Note: This is a Jetpack shortcode.
+		if ( ! $this->is_amp_shortcode_available_in_jetpack( self::SHORTCODE_TAG ) ) {
```

Comment:
See notes above for Soundcloud.

Reply:
Sure, applied in https://github.com/ampproject/amp-wp/pull/3678/commits/b64392bf469255d7573e074a8bc0d0d1f1dc14c8